quantlib/experimental/risk/sensitivityanalysis.pyx

```diff
@@ -0,0 +1,76 @@
+include '../../types.pxi'
+
+from cython.operator cimport dereference as deref
+
+from quantlib.handle cimport shared_ptr, Handle
+
+cimport quantlib._quote as _qt
+cimport quantlib.instruments._bonds as _bd
+cimport _sensitivityanalysis as _sa
+cimport quantlib.instruments._instrument as _it
+from libcpp.vector cimport vector
+from libcpp.pair cimport pair
+from quantlib.quotes cimport SimpleQuote, Quote
+from quantlib.instruments.instrument cimport Instrument
+cimport quantlib.instruments.bonds as _bo
+
+
+cdef public enum SensitivityAnalysis:
+    OneSide
+    Centered
+ 
+
+def bucketAnalysis(quotes_vvsq, instruments,
+                    quantity, shift, type):
```

Comment:
type is a reserved word, you should use `type_` or a name closer to what that type really means, e.g. `analysis_type`